web/src/features/moreCast2/components/DataGridColumns.tsx

```diff
@@ -111,7 +112,7 @@ export class DataGridColumns {
 
 const renderGroupHeader = (
   id: string,
-  weatherParam: string,
+  weatherParam: keyof MoreCastParamColors,
```

Comment:
The `weatherParam` variable is used for more than just color, so I'm not sure it makes sense to relate it to color here?
I messed up by not typing this properly when I did the show hide columns work.

Reply:
Renamed to `MoreCastParams`